otx/algorithms/common/tasks/training_base.py

```diff
@@ -473,6 +475,13 @@ def _load_model_ckpt(self, model: Optional[ModelEntity]):
             if model_data.get("anchors"):
                 self._anchors = model_data["anchors"]
 
+            tiling_parameters = model_data.get("config", None).get("tiling_parameters", None)
```

Comment:
What happens if model_data do not contain "config" attribute? Can we get something from None?

Reply:
Good point! I have made some changes.